crates/tako/src/server/core.rs

```diff
@@ -74,6 +74,16 @@ impl CoreRef {
 }
 
 impl Core {
+    #[inline]
+    pub fn split_tasks_workers(&self) -> (&TaskMap, &WorkerMap) {
```

Comment:
Is it a good name? At least in Tokio "split" methods takes "self".

Reply:
Do you have an idea for a better name? The `split` represents a "splitting borrow" (https://doc.rust-lang.org/nomicon/borrow-splitting.html).

Reply:
I do not have:)